MultiYearStudies.R

```diff
@@ -1,44 +1,53 @@
 
-library(skimr)
 
-#duration
-meta_df %>%
-  group_by(Study_number, OtherFactor, Ecosystem_type) %>%
-  mutate(Duration = Study_midyear - min(Study_midyear)) -> meta_time
+#conduct meta-analysis
+#this is the best fitting model of irrigiation studies - not sure if the "/" notation is correct
```

Comment:
Interesting; I don't know.

Reply:
This is a legacy of my sloppy coding, apologies! I've updated the comments in this file to accurately reflect what's on the script and it's role in the project.